doc/ug/getting_started_dv.md

```diff
@@ -45,10 +45,9 @@ Before running any test, the [UVM RAL model]({{< relref "dv_methodology/index.md
 The [DV simulation flow]({{< relref "hw/dv/tools/README.md" >}}) has been updated to generate the RAL model automatically at the start of the simulation.
 As such, nothing extra needs to be done.
 A hook for generating it is already provided in the generated `dv/Makefile`.
-It can be created manually by simply navigating to the `dv` directory and invoking the command:
+It can be created manually by simply invoking the command:
```

Comment:
```suggestion
It can be created manually by invoking the [`regtool`](path-to-regtool-readme):
```

Reply:
Probably drop the "the" too, so "It can be created manually by invoking regtool".

Reply:
Thanks Sri and Rupert, updated as you recommended.